ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -2,7 +2,7 @@
 
 declaration template components/network/core-schema;
 
-final variable QUATTOR_TYPES_NETWORK_LEGACY ?= true;
+final variable QUATTOR_TYPES_NETWORK_LEGACY ?= false;
```

Comment:
@jrha @ned21 @jouvin i suppose we should give sites one or 2 weeks time to compare generated profiles with changing this boolean.

Reply:
👍  @aka7 is probably the best person to do that for MS. We have pretty good tools for this so it should be easy to generate, but may take a little bit to analyse any differences.

Reply:
@jrha  thank you for this.

I plan to look into this next week. 
I suspect backward incompatibility will be with nm_clean_inactive_conn too.